docker/Dockerfile

```diff
@@ -0,0 +1,67 @@
+# See the file "COPYING" in the main distribution directory for copyright.
+
+# Layer to build Zeek. {{{
+FROM debian:buster-slim AS build_zeek
+
+# Configure system for build.
+RUN apt-get -q update \
+ && apt-get install -q -y --no-install-recommends \
+     bind9 \
+     bison \
+     cmake \
+     flex \
+     g++ \
+     gcc \
+     libmaxminddb-dev \
+     libpcap-dev \
+     libssl-dev \
+     libz-dev \
+     make \
+     python3-minimal \
+     python3-dev \
+     swig \
+     ninja-build \
+     python3-pip \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+COPY . /src/zeek
+WORKDIR /src/zeek
+RUN ./configure \
+     --generator=Ninja \
+     --build-type=Release \
+ && ninja -C build install
+
+# Install dependencies for zkg.
+RUN pip3 install --no-cache-dir GitPython semantic-version
```

Comment:
Just a suggestion: we could also handle these via Debian packages — they are `python3-git` and `python3-semantic-version`, I believe.